app/models/user.rb

```diff
@@ -3,8 +3,16 @@ class User < ApplicationRecord
   has_many :tests_users
   has_many :tests, through: :tests_users
 
+  scope :by_level, -> (level) { joins(:my_test).where(tests: {level: level}) }
+
+  validates :email, presence: true
+
   def show_list_tests_by_level(level)
-    Test.joins("INNER JOIN users on (tests.owner_id = users.id)").where(tests: {level: level})
+    my_test.where(level: level).pluck(:title)
+  end
```

Comment:
I think here should be 'my_testS', because you can't call where for instance. Or if not, here should be something else, chat me for this))